job-server/src/spark.jobserver/WebApi.scala

```diff
@@ -481,8 +482,12 @@ class WebApi(system: ActorSystem,
                     val jobManager = getJobManagerForContext(contextOpt, contextConfig, classPath)
                     val events = if (async) asyncEvents else syncEvents
                     val timeout = timeoutOpt.map(t => Timeout(t.seconds)).getOrElse(DefaultSyncTimeout)
+                    var sJobConfig = jobConfig.withValue("snappydata.user", ConfigValueFactory
+                        .fromAnyRef(authInfo.user.login))
+                    sJobConfig = sJobConfig.withValue("snappydata.password", ConfigValueFactory
+                        .fromAnyRef(authInfo.user.token))
```

Comment:
@ashetkar Isn't this property "snappydata.store.user" for smart connector mode? I think we should make the two consistent to avoid confusion and use "snappydata.user"/password throughout.
Btw, why are this special settings required into jobConfig? Are you getting those in the job implementation and then setting on session in the SnappyJobs?